src/containers/CourseBlockContainer.jsx

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+import CourseBlock from '../components/CourseBlock';
+
+const findCourseById = (allCourses, courseId) => (
+  // eslint-disable-next-line no-underscore-dangle
```

Comment:
we're going to have merge conflicts with the linter-fix branch when that goes up. maybe we don't need these? just for the sake of avoiding them, since they're gonna get inevitably deleted anyways.